src/js/components/ExtraInfo/extraInfo.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ExtraInfo from './index';
+
+const props = {
+  data: '',
+  view: '',
+};
+
+describe('ExtraInfo', () => {
+  it('should be defined', () => {
+    expect(ExtraInfo).toBeDefined();
+  });
+  it('should render correctly', () => {
+    const wrapper = shallow(
+      <ExtraInfo {...props} /> );
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('should render right values in POSTER view', () => {
+    const props = {
+      data: 'Drama',
+      view: 'POSTER',
+    };
+    const wrapper = shallow(
+      <ExtraInfo {...props} /> );
+
+    expect(wrapper.find('.extra-info__content')
```

Comment:
this formatting looks like `.text` is chained to `expect` call, what do you think about:
```expect(wrapper.find('.extra-info__content').text())```?